src/switchroot/ostree-prepare-root.c

```diff
@@ -156,13 +156,43 @@ get_aboot_root_slot (const char *slot_suffix)
   return NULL;
 }
 
+static bool
+read_proc_cmdline_key_starts_with (const char *key)
```

Comment:
I had to read the source of this to understand what it does.  Perhaps something more like (also with a doc comment):

```
// Returns `true` if any kernel argument exists that begins with the provided prefix.
static bool
proc_cmdline_has_key_starting_with (const char *prefix)
```

Reply:
I did starts_with because I've seen that in more than one API:

```
$ grep -r starts_with /usr/include/ | wc -l
51
$ grep -r starting_with /usr/include/ | wc -l
1
```

but meh, this more verbose name is fine too, changing...